src/copa_game/ports/http_in.clj

```diff
@@ -1,5 +1,6 @@
 (ns copa-game.ports.http-in
-  (:require [copa-game.ports.http-out :as http-out]))
+  (:require
+   [copa-game.ports.http-out :as http-out]))
```

Comment:
You can add
```
{:clean {:ns-inner-blocks-indentation :same-line}}
```
to `.lsp/config.edn` to always keep require indentation in the same line

Reply:
Unless you prefer to leave them on a line below. That's fine too! In this case just ignore my comment.

Reply:
Add configuration to .lsp